sdk/storage/azure-storage-blobs/inc/azure/storage/blobs/blob_responses.hpp

```diff
@@ -29,54 +29,153 @@ namespace Azure { namespace Storage {
 
     namespace Models {
 
+      /**
+       * @brief Response type for #Azure::Storage::Blobs::BlobClient::DownloadTo.
+       */
       struct DownloadBlobToResult
       {
+        /**
+         * The blob's type.
```

Comment:
=)

Reply:
maybe add a little info or at least a link to #Models::BlobType so people can click it to see the Type options

Reply:
Yeah, totally we can. I just don't think the comment is necessary. The variable name is fairly self-explanatory.